src/js/lib/monster.ui.js

```diff
@@ -976,60 +979,68 @@ define(function(require) {
 		// Set to 'monthly', it will always set the end date to be one month away from the start date (unless it's after "today")
 		// container: jQuery Object. Container of the datepickers
 		initRangeDatepicker: function(options) {
-			var self = this,
-				container = options.container,
+			var timezone;
+
+			if (_.has(monster, 'apps.auth.currentUser.timezone')) {
+				timezone = monster.apps.auth.currentUser.timezone;
+			} else if (_.has(monster, 'apps.auth.currentAccount.timezone')) {
+				timezone = monster.apps.auth.currentAccount.timezone;
+			} else {
+				timezone = moment.tz.guess();
+			}
+
+			var container = options.container,
 				range = options.range || 7,
 				initRange = options.initRange || options.range || 1,
 				inputStartDate = container.find('#startDate'),
 				inputEndDate = container.find('#endDate'),
-				now = new Date(),
-				today = new Date(now.getFullYear(), now.getMonth(), now.getDate(), 0, 0, 0, 0),
+				initDate = moment().tz(timezone),
+				today = moment().tz(timezone).set({hours: 0, minutes: 0, seconds: 0}),
```

Comment:
You can use `momentjs`'s more verbose methods so we can easily understand the intent:
```suggestion
				today = moment().tz(timezone).startOf('day'),
```

Reply:
great, thanks!